INTRODUCTION.md

```diff
@@ -794,6 +794,64 @@ error log is silenced (will only be seen with debug enabled).
 but it is recommended to instead rely on the above heuristics.
 
 
+#### Sparse connections
```

Comment:
under Brokers section above, maybe the first line is better: "On initialization, librdkafka only needs a partial list of brokers (at least one), called the bootstrap brokers".